.env

```diff
@@ -0,0 +1,9 @@
+DB_HOST=localhost
+DB_PORT=3306
+DB_USERNAME=root
+DB_DATABASE=megak-head-hunter-15
+DB_PASSWORD=''
+HMAC_KEY=foanoqwodiodj2j120jdsamdsamdasdas90daspd9e1om2opdmdasdoasdoas
+JWT_SECRET=JWTsecretdlasdjasdposadaspodaspdmasdoasmdmaspdaspdmaspdmpa
+URL=http://localhost:3000
+REGISTRATION_LINK_EXP_TIME_IN_DAYS=7
```

Comment:
Najlepiej, żeby to puste/przykładowe lądowało na Git. A tak to już zawsze będzie w historii Gita.